layouts/shortcodes/asm-protection-page-configuration.md

```diff
@@ -0,0 +1,7 @@
+The blocked requests feature JSON or HTML content. If the [`Accept` HTTP header][19] is pointing to HTML - like `text/html` -, the HTML content is used, otherwise the JSON one is. 
```

Comment:
```suggestion
The blocked requests feature JSON or HTML content. If the [`Accept` HTTP header][19] is pointing to HTML—like `text/html`—, the HTML content is used; otherwise, the JSON one is used. 
```